lib/microservice-call.js

```diff
@@ -117,23 +174,24 @@ module.exports = class MicroServiceCall {
 
 			await SecretFetcher.fetch();
 
-			const { headers, statusCode, statusMessage, body } = await request({
+			const { data, status, statusText, headers } = await axios.request({
 				url: this._getUrlWithEndpointParameters(apiEndpoint, endpointParameters),
 				headers: {
 					...this.getBasicHeaders(),
 					...requestHeaders
 				},
-				body: requestBody,
-				qs,
-				json: true,
-				method: httpMethod
+				data: requestBody,
+				params: qs,
+				method: httpMethod,
+				validateStatus: () => true,
+				responseType: 'json'
 			});
 
 			return {
 				headers,
-				statusCode,
-				statusMessage,
-				body
+				statusCode: status,
+				statusMessage: statusText,
+				body: data || undefined
```

Comment:
No suma nada el `|| undefined`

```suggestion
				body: data
```